sdk/cloudmachine/Azure.Projects/src/Azure.Projects.csproj

```diff
@@ -9,8 +9,13 @@
     <!-- Disable warning CS1591: Missing XML comment for publicly visible type or member -->
     <NoWarn>CS1591</NoWarn>
     <NoWarn>OPENAI001</NoWarn>
+    <NoWarn>AZC0007</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\core\System.ClientModel\src\System.ClientModel.csproj" />
```

Comment:
Can we file an issue and reference it in a TODO comment to switch back to package ref?